updater/library/src/config.rs

```diff
@@ -59,27 +59,27 @@ impl ResolvedConfig {
     }
 }
 
-pub fn set_config(config: AppConfig) {
+pub fn set_config(config: AppConfig, yaml: YamlConfig) {
     // If there is no base_url, use the default.
     // If there is no channel, use the default.
     let mut lock = global_config()
         .lock()
         .expect("Failed to acquire updater lock.");
-    lock.base_url = config
+    lock.base_url = yaml
         .base_url
         .as_deref()
         .unwrap_or(DEFAULT_BASE_URL)
         .to_owned();
-    lock.channel = config
+    lock.channel = yaml
         .channel
         .as_deref()
         .unwrap_or(DEFAULT_CHANNEL)
         .to_owned();
     lock.cache_dir = config.cache_dir.to_string();
-    lock.client_id = config.client_id.to_string();
-    lock.product_id = config.product_id.to_string();
+    lock.product_id = yaml.product_id.to_string();
     lock.base_version = config.base_version.to_string();
     lock.original_libapp_path = config.original_libapp_path.to_string();
     lock.vm_path = config.vm_path.to_string();
     lock.is_initialized = true;
+    info!("Updater configured with: {:?}", lock);
```

Comment:
do we want to keep this log in?

Reply:
Yes, that one was intentional.  Felt good to see what config the updater actually got.